apps/meteor/ee/client/sidebar/footer/SidebarFooterWatermark.tsx

```diff
@@ -1,16 +1,19 @@
-import { Box } from '@rocket.chat/fuselage';
-import { useTranslation } from '@rocket.chat/ui-contexts';
+import { Box, Skeleton } from '@rocket.chat/fuselage';
 import type { ReactElement } from 'react';
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
-import { useIsEnterprise } from '../../../../client/hooks/useIsEnterprise';
+import { useLicense } from '../../../../client/hooks/useLicense';
 
 export const SidebarFooterWatermark = (): ReactElement | null => {
-	const t = useTranslation();
+	const { t } = useTranslation();
 
-	const { isLoading, isError, data } = useIsEnterprise();
+	const { data: { activeModules = [], trial: isTrial = false, tags = [] } = {}, isLoading, isError } = useLicense();
+	const planName = tags[0]?.name ?? 'Community';
 
-	if (isError || isLoading || data?.isEnterprise) {
+	const isWatermarkHidden = !isTrial && activeModules.includes('hide-watermark');
+
+	if (isWatermarkHidden) {
```

Comment:
Do you agree with me, that as you are setting default values for `data`, when there is an error or loading, it will always return here, never reaching the skeleton?